packages/tabs/components/Tabs.tsx

```diff
@@ -14,52 +21,60 @@ injectGlobal`
 }
 
 .react-tabs__tab-list {
```

Comment:
Why not pass our own classname/s generated by emotion?

Reply:
Because react tabs has not that much of configuration possibilities.